docs/getting-started/nas/synology.md

```diff
@@ -91,10 +91,135 @@ This guide describes how to set up PhotoPrism using the new Synology user interf
 ### First Steps
 
 Our [First Steps 👣](../../user-guide/first-steps.md) tutorial guides you through the user interface and settings to ensure your library is indexed according to your individual preferences.
+-->
 
+## Setup using Docker and MariaDB
 
+### Prerequisites
+- Docker is installed (called 'Container Manager' to Synology.)
```

Comment:
So why call it 'Docker' if even Synology don't call it that?  And if "installed" the prerequisite is satisfied.